lib/ssh_scan/database/sqlite_db.rb

```diff
@@ -55,6 +57,17 @@ def find_scan_result(uuid)
         end
         return scans
       end
+
+      def fetch_available_result(socket)
+        uuid = []
+        @db.execute(
+          "select uuid from api_schema where target like ( ? ) and port like ( ? )",
+          [socket[:target], socket[:port]]
+        ) do |row|
+          uuid << row[1]
+        end
+        return uuid
```

Comment:
I think in this interface you're returning the UUID as a string and not as JSON, where as in the other one, you return JSON.  This should be consistent between the two interfaces, if we expect them to be inter-changable.